scripts/deploys/expo-updates/get-runtime-version.js

```diff
@@ -0,0 +1,7 @@
+// scripts/get-runtime-version.js
+const fs = require("fs")
+const path = require("path")
+
+const appJson = JSON.parse(fs.readFileSync(path.resolve(__dirname, "../../../app.json")))
+const version = appJson.expo.runtimeVersion
+console.log(version)
```

Comment:
Maybe a comment would be useful here since we are expecting this output in the other script.
```suggestion
// Keep this last because ...
console.log(version)
```